CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Changes to Calva Power Tools
 
 - Improved README to talk about Clay
 - Fix: [Uninitialized namespace issue when using Clay commands](https://github.com/BetterThanTomorrow/calva-power-tools/issues/3)
+- Fix: [Default Clay keybindings](https://github.com/BetterThanTomorrow/calva-power-tools/issues/5)
+- Fix: [Synchronize Clay commands](https://github.com/BetterThanTomorrow/calva-power-tools/pull/6)
```

Comment:
I use *Fix:* meaning some bug or major issue was fixed. Here I think we're mostly implementing.

```suggestion
- [Default Clay keybindings](https://github.com/BetterThanTomorrow/calva-power-tools/issues/5)
- [Synchronize Clay commands](https://github.com/BetterThanTomorrow/calva-power-tools/pull/6)
```

Could be argued that the namespace thing is also mainly an improvement, but it depends on perspective. 😄